RemoteHealthcare/SimulatorBike.cs

```diff
@@ -73,15 +60,15 @@ public void RunStep(ref int i, ref Stopwatch stopwatch)
         {
             avansBikeData bikeData = new avansBikeData();
             bikeData.Data = GenerateSpeedData(i, stopwatch);
-            Bluetooth.BleBike_SubscriptionValueChanged(bikeData, false);
+            Bluetooth.BleBike_SubscriptionValueChanged(bikeData);
             i++;
         }
 
         private long elapsedTime = 0;
         private byte[] GenerateSpeedData(int i, Stopwatch stopwatch)
```

Comment:
Zie [Style Guide](https://github.com/Ti-2-1-RH-A1/remote-healthcare/blob/development/.github/STYLE_GUIDELINES.md#code-structure) voor fout in code structuur.